gulpfile.js

```diff
@@ -104,4 +100,4 @@ gulp.task('copy', function() {
         .pipe(gulp.dest(paths.dist));
 });
 
-gulp.task('default', ['clean', 'copy', 'scripts', 'styles', 'jade']);
\ No newline at end of file
+gulp.task('default', gulp.series(['copy', 'scripts', 'styles', 'jade']));
```

Comment:
Gulp allows you to run tasks in parallel now, so we must specify this is in series.